keanu-examples/bayesianMethodsForHackers/src/test/java/com/examples/ChallengerDisasterTest.java

```diff
@@ -0,0 +1,29 @@
+package com.examples;
+
+import io.improbable.keanu.vertices.dbl.KeanuRandom;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.within;
+
+public class ChallengerDisasterTest {
+
+    @Before
+    public void setup() {
+        KeanuRandom.setDefaultRandomSeed(1);
+    }
+
+    @Test
+    public void testWhenChallengerDisasterIsRunThenBetaIsSmallAndPositive() {
+        // act
+        ChallengerDisaster.ChallengerPosteriors posteriors = ChallengerDisaster.run();
+
+        System.out.println("mapAlpha " + posteriors.mapAlpha);
+        System.out.println("mapBeta " + posteriors.mapBeta);
+
+        // assert
+        assertThat(posteriors.mapBeta).isCloseTo(0.25, within(0.15));
```

Comment:
These assertions are pretty wide given the size of the asserted value?

Reply:
I've tightened them up a bit but from the plot in the example the posterior looks pretty broad anyways.